Crane/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt

```diff
@@ -192,15 +195,14 @@ private fun MapViewContainer(
     latitude: String,
     longitude: String
 ) {
+    val cameraPosition = remember(map) { LatLng(latitude.toDouble(), longitude.toDouble()) }
```

Comment:
This won't update if `latitude` or `longitude` change 🤔

Reply:
And `map` doesn't make any sense there! Thanks. Let's try to write correct code! Updating...